pkgs/by-name/ni/nitro/package.nix

```
@@ -0,0 +1,44 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  installShellFiles,
+  lib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nitro";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "leahneukirchen";
+    repo = "nitro";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-RabBawI3qXJwsbHpfuSEBM3n0RQ+Rk7kW8tHHcIIJJI=";
+  };
+
+  outputs = [
+    "out"
+    "man"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  # Compile with size optimisations for minimal targets
+  buildFlags = lib.optionalString stdenv.hostPlatform.isStatic "tiny";
+  installPhase = ''
+    installBin nitro nitroctl
+
+    mv halt.8 nitro-halt.8
+    installManPage nitro.8 nitro-halt.8 nitroctl.1
+  '';
```

```suggestion
  installPhase = ''
    runHook preInstall
    installBin nitro nitroctl

    mv halt.8 nitro-halt.8
    installManPage nitro.8 nitro-halt.8 nitroctl.1
    runHook postInstall
  '';
```
phases must remain adjustable